paddle/fluid/framework/variable.h

```diff
@@ -87,9 +87,9 @@ class Variable {
   // parameter of Variable.
   template <typename T>
   struct PlaceholderImpl : public Placeholder {
-    static_assert(
-        IsRegisteredVarType<T>(),
-        "Not registered type. Please register T inside var_type_traits.h");
+    static_assert(IsRegisteredVarType<T>(),
```

Comment:
static_assert never throws to users. It is not necessary to change this line.

Reply:
Done